### ISSUE TITLE
[QUESTION] Conan revisions feature

### ISSUE THREAD

**Issue by Morkxy** [conan-io/docs#1391]
Hello

I am really interested for use the (experimental) revisions feature. And i have read the documentation here:
https://docs.conan.io/en/latest/versioning/revisions.html

In the bottom of the page, it is written "server support: bintray".

But if i test with bintray repository, i don't find any revisions (exactly, only one: "0").... :(

$ CONAN_REVISIONS_ENABLED=1 conan search qt/5.13.0@bincrafters/stable --revisions -r bincrafters
Revisions for 'qt/5.13.0@bincrafters/stable' at remote 'bincrafters':
0 (2019-08-03 02:08:25 UTC)

$ CONAN_REVISIONS_ENABLED=1 conan search freetype/2.10.0@bincrafters/stable -r bincrafters --revisions
Revisions for 'freetype/2.10.0@bincrafters/stable' at remote 'bincrafters':
0 (2019-08-13 20:06:04 UTC)

$ conan remote list
conan-center: https://conan.bintray.com [Verify SSL: True]
bintray: https://api.bintray.com/conan/conan/conan-center [Verify SSL: True]
bincrafters: https://api.bintray.com/conan/bincrafters/public-conan [Verify SSL: True]

Did I miss something ?

Thanks !!

**Comment by memsharded** [comment]
If you are interested in testing revisions, I would recommend trying with the Artifactory CE, which is totally free (you can use it for your private packages too) and implements revisions too.

**Comment by Morkxy** [comment]
Thank you,
Indeed, i already use Artifactory CE :) but sometime i have a trouble with build a same package version (from bintray or others internet repository) because something has been changed in the recipe....

**Comment by memsharded** [comment]
Yes, we will sync with the Bintray team, the main problem I see with that is that package authors would easily reach the quotas, because of storing all revisions, and will need extra maintenance and would end removing old revisions too anyway...
For that purpose it might be better to go to the authors source repo and try to build from there. Or maybe you could cache what you need in your Artifactory CE, download the third party packages you depend on, upload them to your Artifactory, repeat whenever you update from Bintray. Your Artifactory CE will keep the revisions because they will be updated when installed in your local cache.
